### ISSUE TITLE
Connection Interface

### ISSUE THREAD

**Issue by phildree** [amosproj/amos2023ss01-apache-pulsar-ui#42]
## User story
1. As a user
2. I want to provide a hostname
3. So that I can select the Apache Pulsar Endpoint the UI is connecting to

## Acceptance criteria
* Backendinterface that provides the functionality to set/change the Endpoint it should connect to

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
